js-miniapp-sdk/src/modules/chat-service.ts

```diff
@@ -23,6 +23,18 @@ interface ChatServiceProvider {
     id: string,
     message: MessageToContact
   ): Promise<string | undefined>;
+
+  /**
+   * Opens a contact chooser which allows the user to choose a multiple contacts
+   * and then sends the message to the all chosen contacts.
+   * @param message The message to send to contact.
+   * @returns Promise resolves with an array of contact id which were sent the message.
+   * Can also resolve with an empty array in the case that the message was not sent to any contacts, such as if the user cancelled sending the message.
```

Comment:
Could you also add something similar to this line on the docs for `sendMessageToContactId` and `sendMessageToContact`?

Reply:
Do you mean in the sdk? `sendMessageToContactId` and `sendMessageToContact` already have it.

Reply:
Ah, okay. Sorry, I didn't notice it was already there 👍